meta-resin-common/recipes-core/initrdscripts/files/resindataexpander

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+FREESPACE_LIMIT=10
```

Comment:
This was the limit set in the initial script and it's not part of the changes in this PR. But I do agree that I don't think this is needed in the first place. I'll leave it for now as it is in this PR as I don't want to risk introducing new bugs. I'll create an issue though.